controls/variables.tf

```diff
@@ -0,0 +1,79 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "control_library_name" {
+  type        = string
+  description = "The control library name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/^[a-zA-Z0-9_\\s\\-]*$/`."
+}
+
+variable "control_library_description" {
+  type        = string
+  description = "The control library description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "control_library_type" {
+  type        = string
+  description = "The control library type. Constraints: Allowable values are: `predefined`, `custom`."
+}
+
+variable "version_group_label" {
+  type        = string
+  description = "(Optional) The version group label. Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/`."
+}
+
+variable "latest" {
+  type        = bool
+  description = "(Optional) The latest version of the control library."
```

Comment:
Optional? No such thing with a boolean

Reply:
yeah, its not optional actually. Just took these descriptions from their docs. They need to update this from optional https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/scc_control_library#latest